### ISSUE TITLE
[Deeplab] mobilenet_v2 custom object segmentation errors/warnings

### ISSUE THREAD

**Issue by fiorano10** [tensorflow/models#7413]
### System information
- **What is the top-level directory of the model you are using**: Deeplab
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: Nope
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 16.04
- **TensorFlow installed from (source or binary)**: Installed from source
- **TensorFlow version (use command below)**: 1.12
- **Bazel version (if compiling from source)**:
- **CUDA/cuDNN version**: Both 9.0 and 10.0
- **GPU model and memory**: 1050Ti Max Q and RTX 2080Ti
- **Exact command to reproduce**:
`python deeplab/train.py \
       --logtostderr \
       --training_number_of_steps=300 \
       --train_split="train" \
       --model_variant="mobilenet_v2" \
       --atrous_rate=6 \
       --atrous_rate=12 \
       --atrous_rate=18 \
       --atrous_stride=16 \
       --decoder_output_stride=4 \
       --train_crop_size=416,416 \
       --train_batch_size=4 \
       --dataset=${object_segmentation} \
       --fine_tune_batch_norm=True \
       --base_learning_rate=0.0001 \
       --tf_initial_checkpoint=${PATH_TO_CHECKPOINT/ deeplabv3_mnv2_pascal_train_aug/model.ckpt-3000 \
       --train_logdir=${model_output} \
       --dataset_dir=${dataset_dir} \
       --initialize_last_layer=False \
       --last_layers_contain_logits_only=True`

### Describe the problem
I have 160 images to train on and 40 images to validate the model on, not sure if that's enough. I was able to convert hand 'segmented' images to tfrecords using the `build_voc2012_data.py`. I am only trying to segment a single type of object and everything else is 'background', thus I'm guessing the total no. of classes is 3. 2 classes I labelled + 1 class to ignore. When I'm training using the above command, I get increasing total loss while training for 300 steps. 

When I try to train using `mobilenet_v2_1.0_224` ckpt I get index mismatch errors

When I try to eval the new checkpoints, using this command:
`python deeplab/eval.py \
    --logtostderr \
    --eval_split="val" \
    --model_variant="mobilenet_v2" \
    --atrous_rates=6 \
    --atrous_rates=12 \
    --atrous_rates=18 \
    --output_stride=16 \
    --decoder_output_stride=4 \
    --eval_crop_size=416,416 \
    --dataset="object_segmentation" \
    --checkpoint_dir=${ckpt_path} \
    --eval_logdir=${eval_dir_path} \
    --dataset_dir=${dataset_path}
`
I get the `Not found: Key aspp1_depthwise/BatchNorm/beta not found in checkpoint` error

### Source code / logs
[Here](https://drive.google.com/drive/folders/1mWXKnT9lLwvriuFYLzBmsI-6bdLD57ok?usp=sharing) is the `tfrecord`, `train_log`, `eval_log`, sample images, output of the trained model.

Similar issues #3992

**Comment by gadagashwini** [comment]
Hi @fiorano10,
Check you have used the same flags during both training and evaluation. 
Kindly check similar issue [here](https://github.com/tensorflow/models/issues/4531#issuecomment-410293450). Thank you!
